src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type Props = {
+  title: string;
+};
```

Comment:
```suggestion
type Props = {
  todo: Todo;
};
```
<img width="326" alt="image" src="https://github.com/user-attachments/assets/ab66770d-b114-4fb8-b594-15adf42ae3bf">